client/components/command-palette/use-command-filter.tsx

```diff
@@ -0,0 +1,22 @@
+export const MAGIC_SEPARATOR = '||';
```

Comment:
Not a strong opinion:

I think we would need a separator a bit more singular.
Something like... `|~~~|`

`MAGIC_SEPARATOR` is good. I was also thinking on other names like `COMMAND_SEPARATOR` `COMMAND_FILTER_SEPARATOR` or `FILTER_SEPARATOR`.

Reply:
Renamed to `COMMAND_SEPARATOR` to make it more clear